site/content/docs/howto/supervisor/configure-supervisor-with-azuread.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Configure the Pinniped Supervisor to use Azure Active Directory as an OIDC provider
+description: Set up the Pinniped Supervisor to use Azure Active Directory login.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: With Azure AD
+    weight: 80
+    parent: howto-configure-supervisor
+---
+The Supervisor is an [OpenID Connect (OIDC)](https://openid.net/connect/) issuer that supports connecting a single
+"upstream" identity provider to many "downstream" cluster clients.
+
+This guide shows you how to configure the Supervisor so that users can authenticate to their Kubernetes
+cluster using their Azure Active Directory credentials.
+
+## Prerequisites
+
+This how-to guide assumes that you have already [installed the Pinniped Supervisor]({{< ref "install-supervisor" >}}) with working ingress,
+and that you have [configured a FederationDomain to issue tokens for your downstream clusters]({{< ref "configure-supervisor" >}}).
+
+## Create an Azure AD Application
+
+If you don't already have an Azure subscription, [create a free account](https://azure.microsoft.com/en-us/free/).
+Next, [create a new tenant](https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/create-new-tenant) in
+Azure Active Directory.  This tenant represents your organization.
+
+For example, to create a tenant:
+
+1. In the [Azure portal](portal.azure.com), navigate to _Home_ > _Azure Active Directory_.
+1. Create a new tenant:
+   1. Click `Manage Tenants`.
+   1. Click `Create`.
+   1. Fill out your organization details.
+1. Optionally, just use the `Default Directory` that is already created.
+1. Users can be added to the directory via the  `Manage` > `Users` link.
+1. Create a new app:
+   1. Click `App Registrations`.
+   1. Click `New Registration`.
+   1. Enter a `user-facing display name`.
+   1. Choose supported account types.
+   1.  Enter the `Redirect URI`. The redirect uri will be the `spec.issuer` you configured in your `FederationDomain` appended with `/callback`.
+      1. This step can optionally be done later, however login will not work until a valid URL is provided.
+      1. There is no `Front Channel Logout URL` for the Pinniped Supervisor, so again use `spec.issuer` and append with `/logout`.
```

Comment:
Is this field required? Should it be left blank?

Reply:
You can't leave it blank, unfortunately.  But we also don't support it. This seemed to be the most reasonable workaround.